Mathlib/GroupTheory/GroupAction/MultipleTransitivity.lean

```diff
@@ -284,6 +292,114 @@ end Higher
 
 end MulAction
 
+namespace Equiv.Perm
+
+variable {α : Type*} [Fintype α]
+
+-- Move?
```

Comment:
This comment should be addressed, or removed.

Reply:
Do you believe this pretransivity should be proved here, or in a less specialized file?